include/nuttx/net/mii.h

```diff
@@ -841,6 +845,17 @@
 #define MII_DP83848C_FHF_INT_EN       (1 << 1)  /* Enable Interrupt on False Carrier Counter Register half-full event. */
 #define MII_DP83848C_RHF_INT_EN       (1 << 0)  /* Enable Interrupt on Receive Error Counter Register half-full event. */
 
+/* Atheros AR8031 MII ID1/2 register bits */
+
+#define MII_PHYID1_AR8031            0x004d    /* ID1 value for AR8031 */
+#define MII_PHYID2_AR8031            0xd074    /* ID2 value for AR8031 */
+
+#define MII_AR8031_PSSR_SPEEDMASK    (0x3 << 14) /* Bit 14-15: Speed */
+#define MII_AR8031_PSSR_10MBPS       (0x0 << 14)
+#define MII_AR8031_PSSR_100MBPS      (0x1 << 14)
+#define MII_AR8031_PSSR_1000MBPS     (0x2 << 14)
```

Comment:
```suggestion
#define MII_AR8031_PSSR_SPEEDMASK    (3 << 14) /* Bit 14-15: Speed */
#define MII_AR8031_PSSR_10MBPS       (0)
#define MII_AR8031_PSSR_100MBPS      (1 << 14)
#define MII_AR8031_PSSR_1000MBPS     (2 << 14)
```
just to be consistent with defines above

Reply:
Done.